classes/player/character_sprite.gd

```diff
@@ -180,11 +178,6 @@ func _physics_process(_delta):
 						
 						# Offset origin's X less at the start of the spin. (Looks better!?)
 						position.x *= parent.pound_spin_factor
```

Comment:
This line seems to cause a slight animation hiccup when the spin ends.

Reply:
Confirmed, but probably not related. Was planning to fix in a separate PR.